src/Orchard.Web/Modules/Orchard.Alias/IAliasService.cs

```diff
@@ -18,8 +18,8 @@ public interface IAliasService : IDependency {
         IEnumerable<string> Lookup(RouteValueDictionary routeValues);
         IEnumerable<string> Lookup(string routePath);
 
-        void Replace(string aliasPath, RouteValueDictionary routeValues, string aliasSource);
-        void Replace(string aliasPath, string routePath, string aliasSource);
+        void Replace(string aliasPath, RouteValueDictionary routeValues, string aliasSource, bool isManaged = false);
+        void Replace(string aliasPath, string routePath, string aliasSource, bool isManaged = false);
 

```

Comment:
I'm not sure it makes sense to have the `isManaged` parameter here. Once created an alias would always be either managed or unmanaged. Also I see a risk of this being an optional parameter, very easy to inadvertently change a managed alias to an unmanaged one by simply forgetting to provide this parameter in the call.